src/visualization/context/pagination.tsx

```diff
@@ -59,17 +59,17 @@ export const PaginationProvider: FC<PaginationProviderProps> = ({
     } else {
       setOffset(offset + size)
     }
-  }, [offset, size, setOffset])
+  }, [offset, size, setOffset]) // eslint-disable-line react-hooks/exhaustive-deps
 
   const previous = useCallback(() => {
     setOffset(calcPrevPageOffset(offset, size))
-  }, [offset, size, setOffset])
+  }, [offset, size, setOffset]) 
 
   const setPage = useCallback(
     (page: number) => {
       setOffset(calcOffset(page, maxSize, total))
     },
-    [offset, maxSize, setOffset]
+    [offset, maxSize, setOffset] // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Disabling lint ruling here because the warning is asking for `total` to be included. `total` is the total number of rows in the data, and that does not change. See previous comment explaining why not having `result?.table?.length` as a dependency is fine. 

![image](https://user-images.githubusercontent.com/39343323/128388216-8e5ca299-e298-421c-91e5-946d6079110d.png)